lia-core/src/main/java/com/lithium/community/android/auth/LiLoginActivity.java

```diff
@@ -117,6 +124,14 @@ public void onCreate(Bundle savedInstanceBundle) {
                 webSettings.setJavaScriptEnabled(true);
                 webSettings.setCacheMode(WebSettings.LOAD_NO_CACHE);
                 webSettings.setAppCacheEnabled(false);
+                webSettings.setAllowContentAccess(true);
```

Comment:
Just one question.. since we are giving a lot of access/permissions to a javascript enabled webview.. this makes it vulnerable to XSS.. I hope we are loading validated urls and it is guarded from loading of a malicious url. We recently worked on something similar https://jira.dev.lithium.com/browse/LSWE-5797 / https://jira.dev.lithium.com/browse/INFOSEC-1415

Reply:
We only route curated oauth URL, Chances of unwanted scriting getting executed are highly unlikely, now that permissions model what can be read or executed through scripting I think it is alright.